src/content/guides/advanced-coredns-configuration/index.md

```diff
@@ -38,64 +38,67 @@ Above setting increases the TTL to 60 seconds.
 
 The cache plugin also supports much more detailed configuration which is documented in the [upstream documentation](https://coredns.io/plugins/cache/).
 
-## Additional proxies
+## Additional forwards (formerly known as proxy) {#additional-forwards}
 
-The default proxy entry we set in CoreDNS is
+In CoreDNS version `1.4.0` the proxy plugin has been deprecated. The same behaviour can be achieved now with forward although syntax can be a bit different. The forward plugin indeed has better performance because it reuses opened upstream connections.
```

Comment:
```suggestion
In CoreDNS version `1.4.0` the proxy plugin has been deprecated. The same behaviour can be achieved now with forward although the syntax can be a bit different. The forward plugin has better performance because it reuses opened upstream connections.
```